### PR TITLE
Updated Installations in ReadMe.md

### LINKED ISSUE #445: Update the README.md file. (Add extra prerequisites to download MinGW)
In the Readme file, It is mentioned that you only need to install `HUGO`. 
but when the developer will move forward he/she will have to run `make hugo` and `make` command is not supported by Windows(No idea of other operating systems).

So, there should be an extra dependency to download [MinGW - Minimalist GNU for Windows](https://sourceforge.net/projects/mingw/).
or they follow [this](https://www.youtube.com/watch?v=taCJhnBXG_w) video.

### PR DESCRIPTION
**Pull request by Uttam-Singhh**
Resolves #445 
Updated Installations in ReadMe File, Added `python gen_config.py `

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ git submodule update --init
 The development web server is started with:
 
 ```bash
-make hugo
-hugo server
+make hugo 
+#or
+python gen_config.py
```
Comment (rgommers) [inline README.md:29]: This looks very confusing. Why not leave the existing instructions alone, and add below it:

_If you don't have `make` available (e.g., on Windows), use this instead_:
```
python gen_config.py
hugo
hugo server
```

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @Uttam-Singhh. Splitting instructions like that makes the text above it wrong, so I suggested an alternative.

File: README.md (hunk 1 above)
Comment (Uttam-Singhh) [reply README.md:29]: oh yes! This is actually better I will do the changes asap!

Comment (Uttam-Singhh) [reply README.md]: Done! @rgommers

**Review by rgommers — APPROVED** [review]
LGTM now, thanks @Uttam-Singhh

**Pull request merged**